slsim/lens_pop.py

```diff
@@ -149,6 +151,23 @@ def __init__(
                 kwargs_variability_model=kwargs_variability,
             )
             self._source_model_type = "point_plus_extended"
+        elif source_type == "supernovae_plus_galaxies":
+            from slsim.Sources.point_plus_extended_source import PointPlusExtendedSource
+
+            self.path = os.path.dirname(__file__)
+            new_path = self.path + "/Sources/SupernovaeData/supernovae_data.pkl"
```

Comment:
this is a very specific hardcoded file (how big is it?). Can't this be external, or at least have an option to be external?

Reply:
This is temporary. In future, we will replace this with the supernovae class.